users/models.py

```diff
@@ -1,3 +1,12 @@
+from django.contrib.auth.models import AbstractUser
 from django.db import models
 
+class User(AbstractUser):
+    is_volunteer = models.BooleanField(default=False)
+    is_ngo = models.BooleanField(default=False)
+    is_business = models.BooleanField(default=False)
+
+class Volunteer(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, primary_key=True)
```

Comment:
I guess Volunteer class will have more volunteer-related tables? Might be worth checking @baffour1999 's PR.